test/integration/minimal/minimal_test.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package minimal
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/gcloud"
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/tft"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMinimalModule(t *testing.T) {
+
+	const name = "test-minimal"
+	const region = "us-east1"
+	const hostIpRegEx = "^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])$"
+
+	memStoreT := tft.NewTFBlueprintTest(t)
+
+	memStoreT.DefineVerify(func(assert *assert.Assertions) {
+		memStoreT.DefaultVerify(assert)
+
+		project_id := memStoreT.GetStringOutput("project_id")
+		op := gcloud.Run(t, fmt.Sprintf("redis instances describe %s --project %s --region %s", name, project_id, region))
+
+		assert.Contains(op.Get("locationId").String(), region, "Memomystore instance's GCE region is valid")
+		assert.Regexp(hostIpRegEx, op.Get("host").String(), "Memomystore instance has a valid host IP")
+		assert.Contains(op.Get("name").String(), name, "Memomystore instance has a valid id")
+		assert.Positive(op.Get("memorySizeGb").Int(), "Memorystore instance has been allocated memory")
```

Comment:
Looks like we used to compare equality https://github.com/terraform-google-modules/terraform-google-memorystore/blob/2b01efdba2b2d060e795f6ff102d10566b5cff5a/test/integration/minimal/controls/gcloud.rb#L54

Reply:
It's hard to read Ruby but I thought this meant that "it should include an attribute called `memorySizeGb` which should have a valid Integer value. That's why I asserted that the attribute should be a positive value which is addresses both requirements.

Reply:
based on docs it looks like its converting to int but still equality
https://www.geeksforgeeks.org/ruby-integer-to_i-function-with-example/

Reply:
Changed